src/main/java/juja/sqlcmd/Application.java

```diff
@@ -22,21 +23,139 @@ public static void main(String[] args) {
 
     public void simpleSQL() throws ClassNotFoundException, SQLException {
         connection = getConnection();
+        deleteTable("user");
         System.out.println(getTablesName());
         createTable("user");
         System.out.println(getTablesName());
+        addUsers(users);
+        showTableData("user");
+        changeUserName("user2", "userсhange1");
+        showTableData("user");
+        deleteUser("user3");
+        showTableData("user");
+        connection.close();
+    }
+
+    /**
+     * Method deletes the user with name "userName" from the table "user".
+     *
+     * @param userName the username you want to delete.
+     * @throws SQLException if a database access error occurs
+     */
+    private void deleteUser(String userName) throws SQLException {
+        if (userName == null) {
+            System.err.println("Username is null.");
+        }
+        try (Statement statement = connection.createStatement()) {
+            String query = String.format("DELETE FROM \"user\" WHERE name = '%s'", userName);
+            statement.executeUpdate(query);
+        }
+    }
 
+    /**
+     * Method changes the username in the "name" field in the "user" table.
+     *
+     * @param oldName old user name. It will be overwritten with the new value.
+     * @param newName new user name value.
+     * @throws SQLException if a database access error occurs
+     */
+    private void changeUserName(String oldName, String newName) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            String query = String.format("UPDATE \"user\" SET name = '%s' WHERE name = '%s'", newName, oldName);
+            statement.executeUpdate(query);
+        }
+    }
 
+    /**
+     * The method prints the table data to the console in the format Field1 | Field2 | ... | FieldN
+     *
+     * @param tableName the name of the table whose data should be output to the console.
+     * @throws SQLException if a database access error occurs
+     */
+    private void showTableData(String tableName) throws SQLException {
+        if (tableName == null) {
+            System.err.println("Can't display data. Table Name is Null");
+            return;
+        }
+        System.out.println(getTableData(tableName));
```

Comment:
What do you think about extracting method into variable?

Reply:
I wanted to have fewer lines of code, but probably this is not always correct.